lib/datasource/go/releases-goproxy.ts

```diff
@@ -72,6 +72,10 @@ const lexer = moo.states({
       push: 'characterRange',
       value: (_: string) => '[',
     },
+    trailingSlash: {
```

Comment:
> The documentation for these values state it matches the logic of `path.Match`, however in reality it's actually a prefix based match.

Do you mean that we need to change the Renovate docs? If so can you please edit our docs in this PR as well?

Reply:
Sorry - I meant that the Go documentation states this is the case - There aren't any docs in Renovate that explain the logic behind `GOPRIVATE`/`GONOPROXY`, they just link through to Go's documentation

Reply:
Oh okay! Thanks for the clarification!